functional_tests.go

```diff
@@ -158,6 +158,11 @@ func cleanupBucket(bucketName string, c *minio.Client) error {
 	return err
 }
 
+func isErrNotImplemented(err error) bool {
+	errResp := minio.ToErrorResponse(err)
```

Comment:
This can be written in even one liner.. 

```
func isErrNotImplemented(err error) bool {
         return minio.ToErrorResponse(err).Code != "NotImplemented"
}
```

Reply:
done.